projects/sandbox/src/app/progress-tracker/progress-tracker.component.html

```diff
@@ -108,40 +108,21 @@
       </span>
     </ng-container>
   </div>
+
   <ng-container *ngIf="progressData['tier-zero-info']; let tierInfo">
-    <div
-      class="warning-view"
-      *ngIf="
-        (tierInfo['content-tier'] && tierInfo['content-tier'].total) ||
-        (tierInfo['metadata-tier'] && tierInfo['metadata-tier'].total)
-      "
-      [ngClass]="{
-        closed: !warningViewOpen
-      }"
+    <sb-pop-out
+      #tierPopOut
```

Comment:
is this the name or id of the component?

Reply:
It's a local reference used to handle clicks on the title.  It's a more lightweight way of accomplishing what could be done with a ViewChild reference.